test/jdk/ProblemList.txt

```diff
@@ -793,6 +793,7 @@ java/awt/Modal/InvisibleParentTest/InvisibleParentTest.java 8172245 linux-all
 java/awt/Frame/FrameStateTest/FrameStateTest.java 8203920 macosx-all,linux-all
 java/awt/print/PrinterJob/ScaledText/ScaledText.java 8231226 macosx-all
 java/awt/print/PrinterJob/PrintTextTest.java 8148334 macosx-all
+java/awt/font/GlyphVector/TestGlyphVectorLayout.java 8354987 generic-all
```

Comment:
Probably, you want to add [JDK-8355048](https://bugs.openjdk.org/browse/JDK-8355048) to the list of resolved issues, since you include problem-listing in this changeset, which makes perfect sense.